docs/advanced/stack-config.md

```diff
@@ -268,6 +269,201 @@ volumes:
 ---------------------
 ```
 
+<a name="configuration-variables"></a>
+## Configuration Variables
+
+All variables changed below will require at the minimum a project restart `fin restart` so that they can take effect. Additional steps may be required.
+
+### DOCKSAL_NFS_PATH
+
+The location of the folder on the host machine to mount to virtual box. More info](../file-sharing.md)
+
+### IMAGE_SSH_AGENT
+
+`Default: docksal/ssh-agent:1.0`
+
+The Docker image to use for SSH Agent. This contains the keys that are shared amongst the projects.
+
+### IMAGE_VHOST_PROXY 
+
+`Default: docksal/vhost-proxy:1.2` 
+
+Docker image to use as the VHOST Proxy. This forwards all requests to the appropriate container based upon the hostname that is being requested.
+
+### IMAGE_DNS
+
+`Default: docksal/dns:1.0`
+
+Docker image to use for DNS Routing.
+
+### DOCKSAL_LOCK_UPDATES 
+
+When set, this will allow for Docksal to no longer accept updates. This is usually good in combination of `CI=true`.
+
+### DOCKSAL_ENVIRONMENT 
+
+`Default: local`
+
+Allow for environment specific dockal.yml and docksal.env files. Files will be same format `docksal-${DOCKSAL_ENVIRONMENT}.yml` or `docksal-${DOCKSAL_ENVIRONMENT}.env`. These files should not be committed.
```

Comment:
Let's change to this

```
Allow for environment specific `docksal.yml` and `docksal.env` files. When specified `fin` loads additional configuration from `docksal-${DOCKSAL_ENVIRONMENT}.yml` and/or `docksal-${DOCKSAL_ENVIRONMENT}.env`.
```

These files can be committed. `docksal-local` is not intended to be committed but maybe someone has `docksal-ci.*` or similar.